irr/src/CXMeshFileLoader.cpp

```diff
@@ -966,7 +967,7 @@ bool CXMeshFileLoader::parseDataObjectSkinWeights(SXMesh &mesh)
 	u32 i;
 
 	const u32 jointStart = joint->Weights.size();
-	joint->Weights.resize(jointStart + nWeights);
+	joint->Weights.reserve(jointStart + nWeights);
```

Comment:
so where is `joint->Weights` actually resized then?

Reply:
In the loop below. `AnimatedMesh->addWeight(joint)` adds a weight to `joint->Weights`. Amended the commit to add a comment.